ai_edge_torch/generative/examples/stable_diffusion/pipeline.py

```diff
@@ -136,27 +138,36 @@ def run_tflite_pipeline(
   uncond_prompt:
     The prompt not to guide the image generation.
   cfg_scale:
-    Guidance scale of classifier-free guidance. Higher guidance scale encourages to generate
-    images that are closely linked to the text `prompt`, usually at the expense of lower
+    Guidance scale of classifier-free guidance. Higher guidance scale encourages
+    to generate
+    images that are closely linked to the text `prompt`, usually at the expense
+    of lower
     image quality.
   height:
     The height in pixels of the generated image.
   width:
     The width in pixels of the generated image.
   sampler:
-    A sampler to be used to denoise the encoded image latents. Can be one of `k_lms, `k_euler`,
+    A sampler to be used to denoise the encoded image latents. Can be one of
+    `k_lms, `k_euler`,
     or `k_euler_ancestral`.
```

Comment:
again the formatting seems problematic here

Reply:
The formatting looks right to me here. Mind clarifying?